### PR TITLE
Minor typo fixes in pgclirc

### PR DESCRIPTION
**Pull request by anthonydb**
## Description

* Corrected a couple of minor typos in `pgclirc`
* Added name to AUTHORS and noted change in `changelog.rst`

## Checklist

- [x] I've added this contribution to the `changelog.rst`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

- [ ] I installed pre-commit hooks (`pip install pre-commit && pre-commit install`), and ran `black` on my code.
- [x] Please squash merge this pull request (uncheck if you'd like us to merge as multiple commits)

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -11,6 +11,7 @@ Features:
 Bug fixes:
 ----------
 
+* Minor typo fixes in `pgclirc`. (Thanks: `anthonydb`_)
```
Comment (laixintao) [inline changelog.rst:14]: You need to add your name at the end of this file, otherwise, there will be a rst syntax error which will fail the CI.

**Comment by laixintao** [comment]
Thanks a lot ❤️

File: changelog.rst (hunk 1 above)
Comment (anthonydb) [reply changelog.rst:14]: Name added; looks like tests are passing now!

**Comment by amjith** [comment]
:clinking_glasses:

**Pull request merged**